### ISSUE TITLE
calculating shap_values slow - enable multiprocessing

### ISSUE THREAD

**Issue by nibu147** [shap/shap#2002]
Hello, I'm using KernelExplainer to calculate feature importance for a scikit-learn SVR (with sigmoid kernel).

The execution of .shap_values() method is extremely slow, so I'm writing here to find a way to speed it up.

Questions:
1. I noticed that only one core (of my macos) is used. Since I prefer not modifying the nsamples parameter, is there a way to enable multiprocessing?
2. Is there maybe a faster way to compute feature importance with shap?

The code I'm using is:

        model.fit(X, y)
        explainer = shap.KernelExplainer(model.predict, X)
        shap_values = explainer.shap_values(X=X)
        shap.summary_plot(shap_values, X, plot_type="bar")

Many thanks.
